filament/backend/src/metal/MetalHandles.mm

```diff
@@ -1135,7 +1135,7 @@ BufferDescriptor b(nullptr, 0u, [](void* buffer, size_t size, void* user) {
     return MTLStoreActionStore;
 }
 
-id<MTLTexture> MetalRenderTarget::createMultisampledTexture(id<MTLDevice> device,
+id<MTLTexture> MetalRenderTarget::createMultisampledTexture(MetalContext& context,
```

Comment:
Instead of passing `MetalContext&` to this method, let's simply make it non-static.

Reply:
👍  changed to non-static in https://github.com/google/filament/pull/6124/commits/f9354500cfd72223f491a7d1abe9e19ac6d077cf